### DISCUSSION TITLE
Nested Queries

### DISCUSSION THREAD

**Discussion by nabeelrehman2114** [datalust/seq-tickets#1527] [Help]
Is there any way to use nested query something like this `select ActivityId from stream where appName = 'appName' and Id = (select Id from stream where name = 'Name')` ?

**Comment by liammclennan** [comment]
Hi @nineralpha, 
Currently you need two queries to do this.

**Comment by efurban** [comment]
how do we do this in a dashboard query?  could I have two queries at the same time?

**Reply by nblumhardt** [reply]
Hi @efurban; it's likely the same limitation will prevent this being done on a dashboard, but if you send an example of the kind of query and chart you want to generate, we may have some ideas. Cheers!

**Comment by efurban** [comment]
sure , here is an example, output as a pie chart.  thanks. 
```sql
select 
count(*) as Count
from stream
where Environment = 'Production'  
and @EventType = 0x205600AC 
and @Properties['ivp_owspricing_priceupdate'] is not null
and @Properties['ivp_owspricing_priceupdate'].pnl_date = (
    select max(@Properties['ivp_owspricing_priceupdate'].pnl_date) 
    from stream
    where Environment = 'Production'  
    and @EventType = 0x205600AC 
    and @Properties['ivp_owspricing_priceupdate'] is not null
)
group by 
    @Properties['ivp_owspricing_priceupdate'].dealer.dealer_name
    ```

**Reply by nblumhardt** [reply]
I don't think this can be computed in one query/presented as a pie by Seq currently, but we'll take a look at what might be involved - thanks for the example!

**Reply by nblumhardt** [reply]
See also https://github.com/datalust/seq-tickets/issues/2050, which is tracking this.
